main_test.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
Question: Why do we attribute the work to Kubernetes Authors, and not Metal3 Authors or something?

Reply:
I've updated this to 2023. Other than that I saw that kubernetes authors were attributed in main.go so I used the same here. According to my knowledge since we are using their libraries in our code, maybe that could be the reason for this.

Reply:
Legacy staffs , mainly because these were copied from templates from CAPI and CAPI as well as a few other providers which live in  k8s sigs org use the same, I believe we can either remove this or attribute this to correct authors. Have to check with people with knowledge on Licenses if its fine or not.

Reply:
@kashifest to comment after he has asked around.